### PR TITLE
PEP 12: provide templates for the header and sections

### PR DESCRIPTION
**Pull request by brettcannon**
To help people not to forget something.

### CODE REVIEW

File: pep-0012.rst [hunk 1]
```diff
@@ -148,6 +149,25 @@ directions below.
 
 - Send your PEP submission to the PEP editors at peps@python.org.
 
+For reference, here are all of the possible header fields::
```
Comment (gvanrossum) [inline pep-0012.rst]: Most of these have a specific format. Maybe (in a follow-up PR?) we should describe those? Or is there reference documentation for pep2html.py already?

File: pep-0012.rst [hunk 2]
```diff
@@ -580,6 +600,47 @@ should be avoided.  For ordinary text, use ordinary 'single-quotes' or
...
 
     ``literal text: in here, anything goes!``
+    
+    
+Suggested Sections
+==================
+Various sections are found to be common across PEPs and are outlined in
+PEP 1 [1]_. Those sections are provided here for convenience (sans the
+"Copyright" section and the metadata footer).
+::
```
Comment (gvanrossum) [inline pep-0012.rst]: Again I wonder if we could put a sentence in each explaining what goes where?

Comment (brettcannon) [reply pep-0012.rst]: So the only reason I didn't put it in was to minimize coupling with PEP 1, but I'm fine with adding one-line guidance in each section.

File: pep-0012.rst (hunk 1 above)
Comment (brettcannon) [reply pep-0012.rst]: We can put examples in there.

**Comment by brettcannon** [comment]
@gvanrossum PTAL

**Comment by brettcannon** [comment]
@warsaw since your name is on the PEP, any feedback, or can I merge this?

**Pull request merged**